regression-test/suites/workload_manager_p1/basic_workload_group_test.groovy

```diff
@@ -0,0 +1,230 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import groovy.sql.Sql
+import org.apache.commons.math3.stat.StatUtils
+import org.apache.groovy.parser.antlr4.util.StringUtils
+
+import java.util.concurrent.atomic.AtomicBoolean
+import java.util.concurrent.atomic.AtomicInteger
+
+/*
+
+how to use:
+
+third party dependency:
+    1 need install java and groovy
+    2 commons-math3-3.6.1.jar/mysql-connector-java-5.1.38.jar
+
+command to run:
+    groovy -cp "lib/*" basic_workload_group_test.groovy
+    lib contains commons-math3-3.6.1.jar/mysql-connector-java-5.1.38.jar
+
+data:
+    default sql and data comes from clickbench
+
+*/
+
+
+def begin_time = System.currentTimeMillis()
+
+def url = 'jdbc:mysql://127.0.0.1:9030/hits?useSSL=false'
+def username = 'root'
+def password = ''
```

Comment:
This groovy is not a suite, using java and groovy it can run every where.
if it has a dependency on the suite framework, it mean user need a heavy build environment, such as maven and .m2 dir and compile suite framework successfully first,  this is not easy to use.